packages/web/src/scss/components/SplitButton/README.md

```diff
@@ -0,0 +1,84 @@
+# Split Button
+
+The Split Button component groups multiple related actions a user can take, combining a button with additional options like a dropdown menu or tooltips for better usability.
+
+⚠️ It is recommended to always use the same button colors for all buttons inside a Split Button.
+
+Learn more about the [Button][readme-button] component in its documentation.
+
+Simple variant:
+
+```html
+<div class="SplitButton">
+  <button type="button" class="Button Button--medium Button--primary">Button</button>
+  <button type="button" class="Button Button--medium Button--primary">Button</button>
+</div>
+```
+
+## Variant with Dropdown
+
+You can add a Dropdown component inside a Split Button to provide additional actions.
+
+Learn more about the [Dropdown][readme-dropdown] component in its documentation.
+
+```html
+<div class="SplitButton">
+  <button type="button" class="Button Button--medium Button--primary">Button</button>
+  <div class="Dropdown">
+    <button
+      type="button"
+      class="Button Button--medium Button--primary"
+      data-spirit-toggle="dropdown"
+      data-spirit-target="#dropdown-1"
+      aria-expanded="false"
+      aria-controls="dropdown-1"
+      data-spirit-autoclose="true"
+    >
+      More
+      <svg width="24" height="24" aria-hidden="true" class="ml-400">
+        <use xlink:href="/assets/icons/svg/sprite.svg#chevron-down" />
+      </svg>
+    </button>
+    <div class="DropdownPopover" data-spirit-placement="bottom-start" id="dropdown-1">Dropdown content</div>
+  </div>
+</div>
+```
+
+## Variant with Tooltip
+
+You can also add a tooltip to buttons, which is especially useful for buttons that contain only icons.
+
+Learn more about the [Tooltip][readme-tooltip] component in its documentation.
+
+```html
+<div class="SplitButton">
+  <button type="button" class="Button Button--medium Button--primary">Button</button>
+  <div class="Tooltip d-inline-block" data-spirit-element="tooltip">
```

Comment:
Please remove all d-inline-block (here and in the index file, too). It is not needed.